portfolio/src/main/webapp/script.js

```diff
@@ -211,87 +211,99 @@ function InfoControl(controlDiv) {
   controlUI.appendChild(controlText);
 }
 
+function createMap() {
+  let newMap = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 35.5733225, lng: -80.7187387 },
+    zoom: 8,
+    disableDefaultUI: true,
+    styles: [
+      {
+        featureType: "poi",
+        stylers: [{ visibility: "off" }],
+      },
+      {
+        featureType: "transit",
+        stylers: [{ visibility: "off" }],
+      },
+      {
+        featureType: "road",
+        stylers: [{ visibility: "off" }],
+      },
+    ],
+  });
+
+  return newMap;
+}
+
+function drawPaths(map, tropicalStormData, infoControlDiv) {
+  Object.keys(tropicalStormData).forEach((tropicalStormID) => {
+    let tropicalStormPathCoords = [];
+
+    tropicalStormData[tropicalStormID].forEach((dataPoint) => {
+      tropicalStormPathCoords.push({
+        lat: dataPoint["lat"],
+        lng: dataPoint["lon"],
```

Comment:
Should it be ```dataPoint["lng"]```?

Reply:
In this case it's "lon" as that's is the key value in the JSON data file.